hedera-node/cli-clients/build.gradle.kts

```diff
@@ -21,14 +21,6 @@ plugins {
 
 description = "Hedera Services Command-Line Clients"
 
-configurations.all {
-    exclude("javax.annotation", "javax.annotation.api")
-    exclude("com.google.code.findbugs", "jsr305")
-    exclude("org.jetbrains", "annotations")
-    exclude("org.checkerframework", "checker-qual")
-    exclude("org.hamcrest", "hamcrest-core")
-}
-
```

Comment:
This was a leftover. It's no longer needed. The metadata rules we already have take care of this.